maintainers/scripts/update.py

```diff
@@ -12,27 +12,31 @@
 
 class CalledProcessError(Exception):
     process: asyncio.subprocess.Process
+    stderr: Optional[bytes]
```

Comment:
Is there a case where this can be `None`? It seems to me we are only setting it in the one place?